pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -528,6 +528,51 @@ var _ = Describe("Snapshotter", func() {
 						Expect(len(list)).Should(Equal(3))
 					})
 				})
```

Comment:
You can also add a positive test case when there is no error in deleting snapshots.